app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -2,37 +2,38 @@ package com.example.ithaca_transit_android_v2.states
 
 import com.example.ithaca_transit_android_v2.models.Coordinate
 import com.example.ithaca_transit_android_v2.models.Location
+import com.example.ithaca_transit_android_v2.models.LocationType
 
 // Parent Class
 sealed class SearchState(
-    val startLocation: Location = Location("", Coordinate(0.0, 0.0), ""),
-    val endLocation: Location = Location("", Coordinate(0.0, 0.0), "")
-) {}
+    val startLocation: Location = Location(LocationType.BUS_STOP, "", Coordinate(0.0, 0.0), ""),
+    val endLocation: Location = Location(LocationType.BUS_STOP, "", Coordinate(0.0, 0.0), "")
```

Comment:
The problem was that we didn't want to have to instantiate and pass in 2 Location objects for child states like 'SearchLaunchState' and 'InitClickState' that extend SearchState. 
But I think we will be removing those parameters from the parent SearchState class so that we don't initialize empty Locations.